server/trpc/router/sendEmail.ts

```diff
@@ -0,0 +1,19 @@
+import toast from "react-hot-toast";
+import { SendEmailSchema } from "../../../schema/comment";
+import sendEmail from "../../../utils/sendEmail";
+import { protectedProcedure, router } from "../trpc";
+
+export const sendEmailRouter = router({
+
+    send:protectedProcedure
```

Comment:
you can do your try catch here and in the catch 

catch (error) {
        throw new Error("Failed to send email");
        
        that should trigger the onError for your toast